### PR TITLE
BAU-206 Release notes 2.8.1

### PR DESCRIPTION
**Pull request by horakihor**
Add release notes for 2.8.1

### CODE REVIEW

**Review by uddr — APPROVED** [review]

File: docs/modules/ROOT/pages/release-notes.adoc [hunk 1]
```diff
@@ -2,6 +2,14 @@
 :url-cloud-cli-announcement: https://hazelcast.com/blog/announcing-hazelcast-cloud-cli/
 :url-cloud-sdk-announcement: https://hazelcast.com/blog/announcing-hazelcast-cloud-go-sdk/
 
+== April 9, 2021 - v2.8.1
+
+*Fixes:*
+
+- Azure Storage Account is not deleted during cluster deletion
```
Comment (JakeSCahill) [inline docs/modules/ROOT/pages/release-notes.adoc:9]:
```suggestion
- During cluster deletion, Azure storage accounts are now also deleted 
```

Used the same captialization as Microsoft: https://docs.microsoft.com/en-us/azure/storage/common/storage-account-overview

File: docs/modules/ROOT/pages/release-notes.adoc [hunk 2]
```diff
@@ -2,6 +2,14 @@
...
+- Field label not matching the placeholder text
```
Comment (JakeSCahill) [inline docs/modules/ROOT/pages/release-notes.adoc:10]:
```suggestion
- Field labels now match the placeholder text
```

File: docs/modules/ROOT/pages/release-notes.adoc [hunk 3]
```diff
@@ -2,6 +2,14 @@
...
+- Wan Replication fails on Azure clusters
```
Comment (JakeSCahill) [inline docs/modules/ROOT/pages/release-notes.adoc:11]:
```suggestion
- WAN Replication no longer fails on Azure clusters
```

**Review by JakeSCahill — COMMENTED** [review]
I think it's best to write release notes in terms of what now happens #18

**Comment by horakihor** [comment]
Thanks, @JakeSCahill - good idea!

@cangencer, @johnmdesjardins Could I ask you guys to update release notes for the next releases or help me to find the correct person to do that, please? 

It's a bit hard for me to do that as I am not always in context what we want to "share" with users and what we don't want to publish and keep it as internal only.

Also, I've done that before as nobody else was responsible for that but it had to be done for SOC and ISO immediately.

I feel it should be PM or QA responsibilities but I am happy to discuss that. Looking forward to hearing your ideas.

**Comment by johnmdesjardins** [comment]
Hi Ihor, I can work on release notes moving forward starting from next week. This week we have a management offsite for Cloud messaging, so I won't have much time.

**Comment by johnmdesjardins** [comment]
I can review them though.

**Comment by horakihor** [comment]
Thanks @johnmdesjardins , appreciate your help

**Pull request merged**